documentatie/use-cases/Airgap.md

```diff
@@ -0,0 +1,24 @@
+# Airgap
+
+## De applicatie controleert of er een internetverbinding is (zee)
+
+__Niveau:__ gebruikersdoel, zee  (🌊)
+
+### Hoofdscenario en uitbreidingen
+
+1. Regelmatig: de applicatie stelt vast dat er geen internetverbinding is.
+
+__Uitbreidingen:__  
+1a. De server stelt vast dat er een internetverbinding is:  
+&emsp; 1a1. De applicatie blokkeert de invoer en aanpassingen aan de configuratie.  
+&emsp; 1a2. De applicatie toont een paginavullende foutmelding aan alle gebruikers.  
+&emsp; 1a3. De foutpagina verdwijnt zodra het probleem is verholpen.  
+
+1b. Een client stelt vast dat er een internetverbinding is:  
+&emsp; 1b1. De betreffende client stuurt een melding naar de server.  
+&emsp; 1b2. De applicatie toont een waarschuwing aan alle gebruikers met vermelding van de bron van het probleem.  
```

Comment:
Wat wordt hier bedoelt met "de bron van het probleem"? Is het handig om aan alle gebruikers te laten zien dat "invoerstation 16 een internetverbinding heeft"? Moeten alle invoerders dat weten? Iets met "iemand is verdachte tot schuld bewezen is": het is niet gezegd dat de persoon die achter het invoerstation zit die internetverbinding heeft gemaakt. Je wilt niet dat andere invoerders te snel bepaalde conclusies trekken. 

Is het niet handiger / neutraler de acties die de server doet gelijk te stellen met wat er gebeurd bij een internetverbinding op de server. En alleen de coördinator die bron te laten zien?

Reply:
Suggesties verwerkt in [2b51c1b](https://github.com/kiesraad/abacus/pull/1452/commits/2b51c1b51979102071e30efc02d074a979621d38)